.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI pipeline
+on:
+  push:
+    branches: [main, dev]
+  pull_request:
+    branches: [main, dev]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Linter
+        run: npm run lint
+      - name: Run e2e tests && unit tests
+        run: npm run test
```

It would be great to add a coverage report. Super useful for future testing
```suggestion
        run: npm run test
      - name: Upload Code Coverage Report
        if: ${{ !cancelled() }}
        uses: actions/upload-artifact@v4
        with:
          name: coverage-report
          path: coverage/
          retention-days: 30
```

---

ok, makes sense